### PR TITLE
Updating translations keys

### PR DESCRIPTION
**Pull request by devSMelo**
Finalized, last keys have already been translated

### CODE REVIEW

File: locales/pt_BR/pt_BR.json [hunk 1]
```diff
@@ -166,7 +166,7 @@
     "Customize": "Personalizar",
     "Cyprus": "Chipre",
     "Czech Republic": "República Checa",
-    "Côte d'Ivoire": "Côte d'Ivoire",
+    "Côte d'Ivoire": "Costa do Marfim",
```
Comment (andrey-helldar) [inline locales/pt_BR/pt_BR.json:169]: Already trnaslated: [locales/pt_BR/pt_BR.json#L147](https://github.com/Laravel-Lang/lang/blob/master/locales/pt_BR/pt_BR.json#L147)

File: locales/pt_BR/pt_BR.json [hunk 2]
```diff
@@ -290,7 +290,7 @@
     "It looks like you do not have an active subscription. You may choose one of the subscription plans below to get started. Subscription plans may be changed or cancelled at your convenience.": "Parece que você não tem uma assinatura ativa. Você pode escolher um dos planos de assinatura abaixo para começar. Os planos de assinatura podem ser alterados ou cancelados conforme sua conveniência.",
     "Italy": "Itália",
     "Jamaica": "Jamaica",
-    "Jane Doe": "Jane Doe",
+    "Jane Doe": "Maria desconhecida",
```
Comment (andrey-helldar) [inline locales/pt_BR/pt_BR.json:293]: Is it correct that the surname is written with a lowercase letter?

File: locales/pt_BR/pt_BR.json [hunk 3]
```diff
@@ -633,7 +633,7 @@
     "Tonga": "Tonga",
     "Too Many Requests": "Muitas solicitações",
     "total": "total",
-    "Total:": "Total:",
+    "Total:": "total:",
```
Comment (andrey-helldar) [inline locales/pt_BR/pt_BR.json:636]: Why was it changed to lower case?

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/pt_BR/pt_BR.json (hunk 3 above)
Comment (andrey-helldar) [reply locales/pt_BR/pt_BR.json:636]: If the spelling of the word matches the English version, then it must be specified in the [exclusion](https://github.com/Laravel-Lang/lang/blob/master/excludes/pt_BR.php) file.

File: locales/pt_BR/pt_BR.json (hunk 2 above)
Comment (devSMelo) [reply locales/pt_BR/pt_BR.json:293]: I'm sorry, I'm fixing it.

**Comment by devSMelo** [comment]
Certo, agora está correto.

File: excludes/pt_BR.php [hunk 4]
```diff
@@ -87,6 +87,7 @@
     'Tokelau',
     'Tonga',
     'total',
+    'Total'
```
Comment (andrey-helldar) [inline excludes/pt_BR.php:90]: The word `Total:` must be followed by a colon.

https://github.com/Laravel-Lang/lang/blob/master/locales/pt_BR/pt_BR.json#L636

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: excludes/pt_BR.php (hunk 4 above)
Comment (andrey-helldar) [reply excludes/pt_BR.php:90]:
```suggestion
    'Total:'
```

**Review by andrey-helldar — APPROVED** [review]

File: excludes/pt_BR.php (hunk 4 above)
Comment (andrey-helldar) [reply excludes/pt_BR.php:90]:
```suggestion
    'Total:'
```

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Review by andrey-helldar — APPROVED** [review]
Obrigado

**Pull request merged**